CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Added
 - Ruby 2.4.1 testing
 
+## [1.0.1] - 2018-04-23
+- Removed built in handler filtering. See http://bit.ly/sensu-plugin
```

Comment:
This should go under a `### Breaking Changes` header.

Reply:
Also this removes a different type of filtering than just the filtering in the blog post, see other comments but this will need to be updated to indicate such.